apps/settings/lib/SetupChecks/TaskProcessingPickupSpeed.php

```diff
@@ -55,9 +55,9 @@ public function run(): SetupResult {
 		}
 
 		if ($slowCount / $taskCount < self::MAX_SLOW_PERCENTAGE) {
-			return SetupResult::success($this->l10n->t('Task pickup speed is ok in the last {hours} hours.', ['hours' => self::TIME_SPAN]));
+			return SetupResult::success($this->l10n->t('The task pickup speed has been ok in the last {hours} hours.', ['hours' => self::TIME_SPAN]));
```

Comment:
This is a broken plural that does not work for languages with more than 1 plural form:
```suggestion
			return SetupResult::success($this->l10n->n('The task pickup speed has been ok in the last %n hour.', 'The task pickup speed has been ok in the last %n hours.', self::TIME_SPAN));
```
I tried to outline this in the original PR.
Either the 24 needs to be hardcoded, or you need to use proper plurals.